yal/tests/test_askaquestion.py

```diff
@@ -505,13 +544,26 @@ async def test_state_handle_timeout_handles_type_2_yes(
         "fields": {"aaq_timeout_sent": "", "aaq_timeout_type": ""},
     }
 
-    tester.assert_state("state_pre_mainmenu")
+    tester.assert_state("state_yes_question_answered")
 
     tester.assert_num_messages(1)
-    tester.assert_message(
-        "🤖 *So glad I could help! If you have another question, "
-        "you know what to do!* 😉"
+    message = "\n".join(
+        [
+            "*That's great - I'm so happy I could help.* 😊 ",
+            "",
+            "Is there anything that you would change about my answer?",
+            "",
+            "*Reply:*",
+            "1. No changes",
+            "2. Yes, I have a change",
+            "",
+            "-----",
+            "*Or reply:*",
+            BACK_TO_MAIN,
+            GET_HELP,
+        ]
     )
+    tester.assert_message(message)
```

Comment:
After the changes requested this test and the one below will probably fail because they need the `aaq_mock` parameter and you should test that we're sending the correct feedback on the model